monkestation/code/modules/emotes/code/emote.dm

```diff
@@ -313,23 +313,28 @@
 	if(!.)
 		return
 
-	var/datum/action/cooldown/spell/pointed/projectile/spit/spit_action = new(src)
+	if (locate(/datum/action/cooldown/spell/pointed/projectile/spit) in user.actions)
+		to_chat(user, "<B>You already have spit in your mouth!</B>")
+		return FALSE
 
 	if(ishuman(user))
-		var/mob/living/carbon/human/H = user
+		var/mob/living/carbon/human/human_user = user
 		var/hasHead = FALSE
 
-		for(var/obj/item/bodypart/parts in H.bodyparts)
+		for(var/obj/item/bodypart/parts in human_user.bodyparts)
 			if(istype(parts, /obj/item/bodypart/head))
 				hasHead = TRUE
```

Comment:
just move the return for this var being checked here and kill that var

Reply:
Is this better?

```dm
if(ishuman(user))
		var/mob/living/carbon/human/human_user = user

		if(!(human_user.get_bodypart(BODY_ZONE_HEAD)))
			//Aint got no HEAD what da hell
			to_chat(user,"<B>You try to spit but you have no head!</B>")
			return FALSE
```

Reply:
yeah